drivers/hmis/app/graphql/mutations/create_ce_opportunity.rb

```diff
@@ -4,8 +4,10 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# frozen_string_literal: true
+
 module Mutations
-  class CeOpportunityCreate < CleanBaseMutation
+  class CreateCeOpportunity < CleanBaseMutation
```

Comment:
This is fine, it's consistent with our mutation naming. 

Some context: our convention is a awkward because it doesn't group related mutations together alphabetically; it's hard to find them as the number of mutations grow.

If this was greenfield I might advocate more strongly for changing our convention but maybe it's not worth trying to change at this point.

Reply:
Agreed alphabetical grouping is nice. I suggested this because you had the mutation type name and the class name having different names, which I think is confusing for looking up mutations. Changing the mutation field name would be fine with me too, as long as they match!

Reply:
Would we consider starting to make subdirectories within `drivers/hmis/app/graphql/mutations/`? I don't have too strong a preference, but I'm reflecting that I usually navigating by search, or fuzzy finding filenames, rather than navigating the file structure. I think I'm more likely to search `CreateCeOpportunity` since the majority of our mutations follow the `CreateSomething` pattern.

If not then I can change it back to `CeCreateOpportunity` per Theron's preference.

Reply:
Sub directories do sound appealing, maybe worth a brief exploration? Thanks for fixing the class/filename-mismatch!

Reply:
Sounds good! Moving them into the `ce` subdirectory involved giving them their own namespace too, so `Mutations::CreateCeReferral` => `Mutations::Ce::CreateCeReferral`, etc. I think this is OK, but let me know if any objections.

 https://github.com/greenriver/hmis-warehouse/pull/5192/commits/e5dec6bcb500d83523aa8343c1db5084b09ac440